data/json/recipes/other/tool.json

```
@@ -136,9 +136,10 @@
     "category": "CC_OTHER",
     "subcategory": "CSC_OTHER_TOOLS",
     "skill_used": "fabrication",
-    "difficulty": 3,
-    "time": "20 m",
+    "difficulty": 1,
+    "time": "15 m",
```

You're not automatically an "experienced flintknapper" because you have a proficiency.
```suggestion
    "time": "20 m",
```

---

Is the player making a good quality hand axe in 15 minutes, or a simple hand axe in 3 minutes?  We can gauge the time better once we nail down the requirements on knapping.